hbase_storage/src/main/java/com/splicemachine/storage/ClientPartition.java

```diff
@@ -157,6 +157,9 @@ public boolean checkAndPut(byte[] key,byte[] family,byte[] qualifier,byte[] expe
                 }
             }
             List<HRegionLocation> tableLocations = ((ClusterConnection) connection).locateRegions(tableName, !refresh, false);
+            if (outdated(tableLocations)) {
+                tableLocations = ((ClusterConnection) connection).locateRegions(tableName, refresh, false);
```

Comment:
Is it better to use "false" explicitly than the variable "refresh" to force not to use the cache?